airflow/models/mappedoperator.py

```diff
@@ -201,7 +201,7 @@ def _expand(self, **mapped_kwargs: "Mappable") -> "MappedOperator":
         from airflow.operators.empty import EmptyOperator
 
         validate_mapping_kwargs(self.operator_class, "expand", mapped_kwargs)
-        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="mapping already partial")
+        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="unmappable or already specified")
```

Comment:
```suggestion
        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="unmappable or already specified (see https://airflow.apache.org/docs/apache-airflow/stable/concepts/dynamic-task-mapping.html#mapping-with-non-taskflow-operators )")
```

Don't we want to direct the user to the exact place in the docs where more context and details is explained? I think that would be an empathetic bahaviour towards the users who see it for the very first time.

Reply:
We can even create a short link for it: 

```suggestion
        prevent_duplicates(self.kwargs, mapped_kwargs, fail_reason="unmappable or already specified (see https://s.apache.org/airflow-unmappable) ")
```

Reply:
Personally I don’t like putting documentation links in the error message. We could make this more prominent in the documentation page though, perhaps adding a section with the error message in the title.